### PR TITLE
anukulsahu: Task 1, 2, 3 and 4

### PR DESCRIPTION
**Pull request by anukulsahu**
Hey admin! Anukul here, i have completed upto task 3,,baaki baad me karunga

### CODE REVIEW

File: Anukulsahu/task1.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+            background-color: rgb(202, 64, 64);
+            color: rgb(94, 182, 152);
+            width: 400px;
+        }
+    </style>
+</head>
+<body>
+    <h2>This is my heading</h2>
+    <p>Hi admin!! My name is Anukul!! Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquid repellat soluta ratione possimus cumque recusandae impedit obcaecati veritatis, placeat doloremque, odit eos quibusdam laborum dolorem qui quisquam, officiis eum atque pariatur accusamus quidem deleniti cum unde voluptatibus? Itaque, ad libero commodi et alias vitae!</p>
```
Comment (PradyumnaKrishna) [inline Anukulsahu/task1.html:19]: Paragraph is not justified and you have not set `max-width`.

File: Anukulsahu/task3.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        #r4{
+            background-color: yellow;
+        }
+       
+        
+    </style>
+</head>
+<body>
+    <table class="center" >
```
Comment (PradyumnaKrishna) [inline Anukulsahu/task3.html:45]: `max-width` is missing from table cells.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Great start @anukulsahu, but you missed few points. Please fix them.

**Comment by anukulsahu** [comment]
I have corrected the issue mentioned and also completed task4 . Please confirm it.

**Review by PradyumnaKrishna — APPROVED** [review]
Well Done @anukulsahu!
LGTM

**Pull request merged**